nexus-writer/Readme.md

```diff
@@ -0,0 +1,81 @@
+# nexus-writer
```

Comment:
Looks like you need to rebase this PR, the documentation linting is not being picked up for some reason.